framework/src/modules/random/module.ts

```diff
@@ -122,7 +124,7 @@ export class RandomModule extends BaseModule {
 			usedHashOnions.push(nextUsedHashOnion);
```

Comment:
this index logic is also done inside of `setLatest` function?

Reply:
I think then intention was to replace `generatorSubStore.set` and use `setLatest` and remove the duplicate code?